js/contextMenus.js

```diff
@@ -578,26 +569,14 @@ function hamburgerTemplateInit (location, e) {
         CommonMenu.separatorMenuItem,
         CommonMenu.importBookmarksMenuItem()
       ]
-    }, {
-      label: locale.translation('bravery'),
-      submenu: [
-        CommonMenu.braveryGlobalMenuItem(),
-        CommonMenu.braverySiteMenuItem()
-      ]
     },
     CommonMenu.downloadsMenuItem(),
+    CommonMenu.findOnPageMenuItem(),
+    CommonMenu.printMenuItem(),
     CommonMenu.separatorMenuItem,
-    {
-      label: locale.translation('help'),
-      submenu: [
-        CommonMenu.aboutBraveMenuItem(),
-        CommonMenu.separatorMenuItem,
-        CommonMenu.checkForUpdateMenuItem(),
-        CommonMenu.separatorMenuItem,
-        CommonMenu.reportAnIssueMenuItem(),
-        CommonMenu.submitFeedbackMenuItem()
```

Comment:
We should keep one of them. It promotes teamwork.  :   )

Chrome has Report and Issue. FX has Submit feedback. Any pref @bbondy ?